### ISSUE TITLE
yarn pull fails for tfjs-node-gpu 1.2.10

### ISSUE THREAD

**Issue by caisq** [tensorflow/tfjs#2156]
#### TensorFlow.js version
tfjs-node-gpu 1.2.10 

#### Code to reproduce the bug / link to feature request
Just add `"@tensorflow/tfjs-node-gpu": "1.2.10"` as a dependency in package.json and do `yarn`.

See the following error:

> [4/4] Building fresh packages...
> error /home/cais/tf-scratch/tfjs-node-6/node_modules/@tensorflow/tfjs-node-gpu: Command failed.
> Exit code: 1
> Command: ./prep-gpu.sh
> Arguments: 
> Directory: /home/cais/tf-scratch/tfjs-node-6/node_modules/@tensorflow/tfjs-node-gpu
> Output:
> cp: cannot stat '../tfjs-node/src': No such file or directory

The same workflow works fine for tfjs-node 1.2.10 (CPU version) and for tfjs-node-gpu 1.2.9.

**Comment by kangyizhang** [comment]
This is a bug in tfjs-node-gpu@1.2.10 duplicate of https://github.com/tensorflow/tfjs/issues/2150. I'll release tfjs-node-gpu@1.2.11 once the fix is merged.
